packages/queued-request-controller/package.json

```diff
@@ -66,7 +66,7 @@
   },
   "peerDependencies": {
     "@metamask/network-controller": "^18.0.0",
-    "@metamask/selected-network-controller": "^10.0.0"
+    "@metamask/selected-network-controller": "^11.0.0"
```

Comment:
This is a breaking change. It should be referenced in the changelog, and represented with a minor version bump (since this is pre-v1)

Reply:
Ah my bad, I suggested that this be a patch bump to Alex in a DM, and overlooked this.

Reply:
fixed here: https://github.com/MetaMask/core/pull/4111/commits/80987b72047c72b67b96086e63d48fa290537a0d